### DISCUSSION TITLE
ARM64 ???

### DISCUSSION THREAD

**Discussion by madabu977** [Pennyw0rth/NetExec#241] [Q&A]
hi everybody. 

its just me or we are missing netexec for arm?

**Comment by NeffIsBack** [comment]
Doesn't python/pipx work on arm?

**Reply by madabu977** [reply]
yo! that's what I call quick response... thanks thought 

Easy answer to your question: yes! Python / pip/ pix all them work on arm… that wasn’t the issue

All start this morning when I was watching ippsec `rebound` video (f!@#% awesome content as usual btw) . I'm used to use `cme` and I was ok let’s give `nxc` a try. At first couldn’t even installed but after a couple of hours lost in the rabbit hole and some troubleshooting I found the problem. Now it’s fix and running.

Will keep this post for a bit in case someone needs help… will delete after
